tox.ini

```diff
@@ -16,6 +16,7 @@ passenv =
     PYTHONPATH
 deps =
     cairo-lang==0.8.1
+    cairo-nile
```

Comment:
should we fix versions in here to avoid breaking changes impacting this repo's development?

Reply:
Yes, good idea